### ISSUE TITLE
Error: Inara setCommanderReputationMinorFaction, The reputation value exceeds the valid range

### ISSUE THREAD

**Issue by louisrousseau** [EDCD/EDMarketConnector#475]
Running 3.4.3.0, I have gotten this error message in the status bar, did not notice exactly when it did so unsure what conditions might trigger it for the moment.

Inara setCommanderReputationMinorFaction, The reputation value exceeds the valid range

**Comment by Wikibear** [comment]
I can confirm this issue.

**Comment by TheInfamousAlk** [comment]
Can confirm this issue as well.

**Comment by ZTiKnl** [comment]
same here, and here is the corresponding part of EDMarketConnector.log

`Inara	202 Everything was alright, the near-neutral status just wasn't stored.	{"eventName": "setCommanderReputationMinorFaction", "eventTimestamp": "2020-02-18T16:27:00Z", "eventData": [{"minorfactionName": "Mooramba aristocrats", "minorfactionReputation": 0.0}, {"minorfactionName": "Vagha Travel Commodities", "minorfactionReputation": 0.0}, {"minorfactionName": "Sirius Hyperspace", "minorfactionReputation": 0.0}, {"minorfactionName": "Cail Autocracy", "minorfactionReputation": 0.0}, {"minorfactionName": "Pilots' Federation Local Branch", "minorfactionReputation": 75.664299}, {"minorfactionName": "Leleng Musu Gold Family", "minorfactionReputation": 0.0}, {"minorfactionName": "Integrated Mooramba", "minorfactionReputation": 0.0}, {"minorfactionName": "Massagetae Blue Camorra", "minorfactionReputation": 0.0}]}`

**Comment by ZTiKnl** [comment]
I recall something from the last (ED) update being another decimal point being added to some type of reputation?
Not sure if this is connected though...

**Comment by ZTiKnl** [comment]
latest journal file (removed most unrelated data):

```
{ 
"timestamp":"2020-02-18T16:33:08Z", 
"event":"FSDJump", 
"StarSystem":"Varam", 
"SystemAddress":5068196095409, 
"Factions":[ 
  { 
    "Name":"Pilots' Federation Local Branch", 
    "Influence":0.000000, 
    "MyReputation":75.664299 
  }, { 
    "Name":"Varam Blue Legal Industries", "Influence":0.093000, "MyReputation":30.860001 
  }, { 
    "Name":"Varam People's Party", "Influence":0.110000, "MyReputation":5.600000
  }, { 
    "Name":"Kachatrimih Jet General Exchange", "Influence":0.110000, "MyReputation":-75.000000
  }, { 
    "Name":"Varam Crimson Organisation", "Influence":0.129000, "MyReputation":0.000000
  }, { 
    "Name":"Defence Party of Varam", "Influence":0.063000, "MyReputation":-2.640000
  }, { 
    "Name":"Liberals of Varam", "Influence":0.034000, "MyReputation":-18.200001 
  }, { 
    "Name":"Independent Defence Agency", "Influence":0.461000, "MyReputation":100.000000
  } 
], 
"SystemFaction":{ "Name":"Independent Defence Agency" }, 
```

**Comment by pwerken** [comment]
I've also been getting these error messages. Looking through my old journal logs the "MyReputation" has always had 6 decimals, so doesn't look like FDev added something like an extra decimal point...

I've collected my Inara 400 error's in a [gist](https://gist.github.com/pwerken/9989b735d36357a30ca3ebdb352f3ba0).
The only recurring element seems to be the "Pilots' Federation Local Branch" minorFaction.
But if that's the problem I would expect to get the error on (nearly) every FSDJump, and that's not the case...

Just spotted that another commander posted a question about this on the [Inara board](https://inara.cz/board/1/315763/#315763).

**Comment by pwerken** [comment]
Have been looking through the [api docs](https://inara.cz/inara-api-docs/#eventprops-45) and I might have found the cause.
It states that the value of "minorfactionReputation" must in the range of [-1..1], while EDMC is passing in the percentage with range [-100..100].
If this is it, there needs to be a divide by 100 added [here](https://github.com/Marginal/EDMarketConnector/blob/master/plugins/inara.py#L377).

**Comment by VAKazakov** [comment]
fix in #479

**Comment by Athanasius** [comment]
Closed as being fixed for #479
